mne_bids/utils.py

```diff
@@ -227,17 +226,25 @@ def _read_events(events_data, event_id, raw, ext, verbose=None):
             raise ValueError('Events must have second dimension of length 3, '
                              f'found {events_data.shape[1]}')
         events = events_data
-    elif 'stim' in raw:
-        events = find_events(raw, min_duration=0.001, initial_event=True,
-                             verbose=verbose)
-    elif ext in ['.vhdr', '.set'] and check_version('mne', '0.18'):
-        events, event_id = events_from_annotations(raw, event_id,
-                                                   verbose=verbose)
     else:
+        events = np.empty(shape=(0, 3))
+
+    # get events from annotations
+    events_annot, event_id_annot = events_from_annotations(raw, event_id,
+                                                           verbose=verbose)
+    if event_id is not None:
+        events = np.concatenate((events, events_annot), axis=0)
```

Comment:
Probably unnecessary, but could you sort based on the first column too?

```suggestion
        events = np.concatenate((events, events_annot), axis=0)
        events.sort(axis=0)
```